src/main/java/Duke/DukeException.java

```diff
@@ -0,0 +1,27 @@
+package Duke;
+
+public class DukeException {
+    void todoException() {

```

Comment:
Interesting use of methods to display exception messages. Since these are methods, their naming should be represented using verbs. throwTodoException() and so on would be better.